components/icon/icon.twig

```diff
@@ -35,12 +35,16 @@
 {% set theme = theme|default('localgov_base') %}
 {% set icon_wrapper_element = icon_wrapper_element|default('div') %}
 
+{%- set icon_fullpath -%}
+  {% if icon_path|first != '@' %}@{{ theme }}/{% endif %}{{ icon_path|default('includes/icons') }}/{{ icon_name }}.svg
```

Comment:
I think we can remove the `{% set ... %}` here then replace line 49 later with line 39 from here.